lib/suspenders/app_builder.rb

```diff
@@ -571,6 +571,15 @@ def configure_quiet_assets
       inject_into_class "config/application.rb", "Application", config
     end
 
+    def configure_lib_directory
+      config = <<-RUBY
+    config.autoload_paths << Rails.root.join('lib')
+    config.eager_load_paths << Rails.root.join('lib')
```

Comment:
i don't normally do this `eager_load_paths` when i'm adding `autoload_paths`, should i?